pkg/config.go

```diff
@@ -39,6 +39,35 @@ func (*Config) Project() string {
 	return os.Getenv("GOOGLE_PROJECT")
 }
 
+// TlsVersion will be used current TLS Policy
+// by this value. The acceptable value MUST BE TLS 1.1/1.2/1.3
+// Note that the TLS 1.3 supports `TLS_AES_128_GCM_SHA256`,
+// `TLS_AES_256_GCM_SHA384` and `TLS_CHACHA20_POLY1305_SHA256` ciphers only.
+// Default is "TLS_1_2"
+func (*Config) TlsVersion() string {
+	tlsVersions := []string{"TLS_1_1", "TLS_1_2", "TLS_1_3"}
+	for _, ver := range tlsVersions {
+		if os.Getenv("MIN_TLS_VERSION") != ver || os.Getenv("MIN_TLS_VERSION") == "" {
+			return "TLS_1_2"
+		}
+	}
+	return os.Getenv("MIN_TLS_VERSION")
+}
+
+// SslProfile returns the string value of pre-configured profile
```

Comment:
The comment should be more clear. Please also double check comment for typos/errors and check recommended comment format mentioned above

Reply:
Fixed by https://github.com/commercetools/gcp-ssl-policy-asserter/pull/3/commits/6f590c87e8855b8392419a2bb64a92a384f2a23c